airflow/www/templates/airflow/dags.html

```diff
@@ -31,15 +31,15 @@
   <meta name="paused_url" content="{{ url_for('Airflow.paused') }}">
   <meta name="status_filter" content="{{ status_filter }}">
   <meta name="autocomplete_url" content="{{ url_for('AutocompleteView.autocomplete') }}">
-  <meta name="graph_url" content="{{ url_for('Airflow.graph') }}">
+  <meta name="graph_url" content="{{ url_for('Airflow.legacy_graph') }}">
   <meta name="dag_run_url" content="{{ url_for('DagRunModelView.list') }}">
   <meta name="task_instance_url" content="{{ url_for('TaskInstanceModelView.list') }}">
   <meta name="blocked_url" content="{{ url_for('Airflow.blocked') }}">
   <meta name="csrf_token" content="{{ csrf_token() }}">
   <meta name="last_dag_runs_url" content="{{ url_for('Airflow.last_dagruns') }}">
   <meta name="dag_stats_url" content="{{ url_for('Airflow.dag_stats') }}">
   <meta name="task_stats_url" content="{{ url_for('Airflow.task_stats') }}">
-  <meta name="tree_url" content="{{ url_for('Airflow.tree') }}">
+  <meta name="tree_url" content="{{ url_for('Airflow.legacy_tree') }}">
```

Comment:
Why do we need to use the legacy urls here instead of the new ones?

Reply:
There are multiple places in the JS code where the dag_id is specified as query parameters. As I've mentioned before, I would much prefer to address it in a separate PR as I imagine that it is not obvious what would be the most correct way to pass Python-generated URL to JS code with view_arg. (And I'd like to keep changes compact, rather than having a full and hard to debug rewrite in a single PR).

Are we doing to insert some templated string like '<:dag_id>'? If yes, how can we avoid breakage if the view_arg is renamed/new view_args added for a path? Or should we avoid passing dynamic URLs over the meta tags all together and come up with something else?

Plus, right now with `legacy_` and `tree` views we will have a clear indication for places that will need to be updated (which I've started working on already in subsequent commit).

Reply:
Ok thats fine. I don't want to expand the scope of this PR too much.